Guide.docc/Swift6Mode.md

```diff
@@ -0,0 +1,69 @@
+# Enabling The Swift 6 Language Mode
+
+Guarantee your code is free of data races by enabling the Swift 6 language mode.
+
+## Using the Swift compiler
+
+To enable complete concurrency checking when running `swift` or `swiftc`
+directly at the command line, pass `-swift-version 6`:
+
+```
+~ swift -swift-version 6 main.swift
+```
+
+## Using SwiftPM
+
+### Command-line invocation
+
+`-swift-version 6` can be passed in a Swift package manager command-line
+invocation using the `-Xswiftc` flag:
+
+```
+~ swift build -Xswiftc -swift-version 6
+~ swift test -Xswiftc -swift-version 6
+```
+
+### Package manifest
+
+To change the language mode for package as a whole:
+
+```swift
+let package = Package(
```

Comment:
maybe worth including the 

```
// swift-tools-version:6.0
// The swift-tools-version declares the minimum version of Swift required to build this package.
```

lines?

Reply:
oh yeah definitely.